src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceCaseMap.cs

```diff
@@ -64,6 +68,39 @@ public override bool RunTask ()
 					AddRename (tok [1].Replace ('/', Path.DirectorySeparatorChar), tok [0].Replace ('/', Path.DirectorySeparatorChar));
 				}
 			}
+			var resmap = ".net/__res_name_case_map.txt";
+			foreach (var aar in AarLibraries ??  Array.Empty<string>()) {
+				Log.LogDebugMessage ($"Processing Aar file {aar}");
+				if (!File.Exists (aar)) {
+					Log.LogDebugMessage ($"Skipping non-existent aar: {aar}");
+					continue;
+				}
+				using (var file = File.OpenRead (aar)) {
+					using var zip = ZipArchive.Open (file);
```

Comment:
Given #9623 and the apparent *immense* performance difference between Xamarin.Tools.Zip and System.IO.Compression, I think we should try using System.IO.Compression to read the `.aar` files, unless we require Xamarin.Tools.Zip for some reason…